emscripten.py

```diff
@@ -2228,7 +2268,15 @@ def debug_copy(src, dst):
     cmd.append('--output-source-map-url=' + shared.Settings.SOURCE_MAP_BASE + os.path.basename(shared.Settings.WASM_BINARY_FILE) + '.map')
   if not shared.Settings.MEM_INIT_IN_WASM:
     cmd.append('--separate-data-segments=' + memfile)
-  if not shared.Settings.SIDE_MODULE:
+  if shared.Settings.SIDE_MODULE:
+    cmd.append('--side-module')
+  else:
+    # Global base is used to calulate the size of the static data segment.  The
+    # value should match the --global-base passed to lld in Building.link_lld()
+    if shared.Settings.RELOCATABLE:
+      cmd.append('--global-base=0')
```

Comment:
the comment doesn't explain why it's 0 in the relocatable case? (and I still don't understand why myself too :)